A/AbInitioSoftwareBase/Compat.toml

```diff
@@ -6,6 +6,7 @@ julia = "1"
 
 ["0-0.2.0"]
 Configurations = "0.3-0.15"
+Pkg = "1"
 
```

Comment:
I don't know exactly what changes this is supposed to make, but only updating compat for the oldest version seems unreasonable.

Reply:
Ok thanks. That was the connection I was missing.